react/README.md

```diff
@@ -21,3 +28,10 @@
 - [Deploy Create React App project to S3](./docs/cra-s3-compressed-deploy.md)
 
 - [Testing guide with Jest & Enzyme](./docs/jest-enzyme-testing.md)
+
+## Posts
+- [Introducing redux-recompose](https://medium.com/wolox-driving-innovation/932e746b0198)
```

Comment:
Also I'd add:
https://medium.com/wolox-driving-innovation/improve-your-react-web-app-with-a-chat-widget-14a1792902df

Reply:
i hadnt seen this 👍